### PR TITLE
[Merged by Bors] - feat(FieldTheory/PolynomialGaloisGroup): The Galois group of an irreducible polynomial acts transitively on the roots

### PR DESCRIPTION
**Pull request by tb65536**
This PR adds a lemma stating that the Galois group of an irreducible polynomial acts transitively on the roots.

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/FieldTheory/PolynomialGaloisGroup.lean [hunk 1]
```diff
@@ -198,6 +198,14 @@ instance galAction [Fact (p.Splits (algebraMap F E))] : MulAction p.Gal (rootSet
     simp only [smul_def, Equiv.apply_symm_apply, Equiv.symm_apply_apply, mul_smul]
 #align polynomial.gal.gal_action Polynomial.Gal.galAction
 
+lemma galAction_isPretransitive [Fact (p.Splits (algebraMap F E))] (hp : Irreducible p) :
```
Comment (erdOne) [inline Mathlib/FieldTheory/PolynomialGaloisGroup.lean:201]: I wonder how hard it would be to upgrade this to an iff.

Comment (tb65536) [reply Mathlib/FieldTheory/PolynomialGaloisGroup.lean:201]: It's not an `iff` (e.g., `x^2`, or more generally `p^2` for any irreducible polynomial `p`).

Comment (erdOne) [reply Mathlib/FieldTheory/PolynomialGaloisGroup.lean:201]: Ah indeed. It needs separability conditions.

**Comment by erdOne** [comment]
maintainer merge

File: Mathlib/FieldTheory/PolynomialGaloisGroup.lean (hunk 1 above)
Comment (alreadydone) [reply Mathlib/FieldTheory/PolynomialGaloisGroup.lean:201]: Inseparability is fine, we just need square-freeness.

**Review by jcommelin — APPROVED** [review]
Thanks :tada:

bors merge

**Pull request closed without merging**